src/main/java/net/dv8tion/jda/api/EmbedBuilder.java

```diff
@@ -690,7 +690,7 @@ public EmbedBuilder setFooter(@Nullable String text, @Nullable String iconUrl)
         }
         else
         {
-            Checks.check(text.length() <= MessageEmbed.TEXT_MAX_LENGTH, "Text cannot be longer than %d characters.", MessageEmbed.TEXT_MAX_LENGTH);
+            Checks.check(text.length() <= MessageEmbed.FOOTER_MAX_LENGTH, "Text cannot be longer than %d characters.", MessageEmbed.FOOTER_MAX_LENGTH);
```

Comment:
Should this also be "Footer cannot be longer than %d characters"?

Reply:
I wondered about this too... I assume it's honestly a copy-paste, but I won't just change this text unless told so.

Reply:
The parameter is called text